test/e2e/specs/site-editor/navigation.spec.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * WordPress dependencies
+ */
+const { test, expect } = require( '@wordpress/e2e-test-utils-playwright' );
+
+test.use( {
+	editorNavigationUtils: async ( { page, pageUtils }, use ) => {
+		await use( new EditorNavigationUtils( { page, pageUtils } ) );
+	},
+} );
+
+test.describe( 'Site editor navigation', () => {
+	test.beforeAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'emptytheme' );
+	} );
+
+	test.afterAll( async ( { requestUtils } ) => {
+		await requestUtils.activateTheme( 'twentytwentyone' );
+	} );
+
+	test( 'Can use keyboard to navigate the site editor', async ( {
+		admin,
+		editorNavigationUtils,
+		page,
+		pageUtils,
+	} ) => {
+		await admin.visitSiteEditor();
+
+		// Test: Can navigate to a sidebar item and into its subnavigation frame without losing focus
+		// Go to the Pages button
+
+		await editorNavigationUtils.tabToLabel( 'Pages', { times: 10 } );
```

Comment:
```suggestion
		await editorNavigationUtils.tabToLabel( 'Pages' );
```

I think it is unnecessary here since the default value is defined on the function side